Adding latest dapper video (#122)
Unity-Technologies/com.unity.multiplayer.docs: docs/learn/dapper-video.md

title: How To Make A Multiplayer Game In Unity 2021.1
---

This page aims to collate a series of Multiplayer Unity Networking tutorial videos that teach various concepts of the new Unity Official Multiplayer solution MLAPI, all made by Dapper Dino. Learn more about Dapper Dino and his [Youtube channel](https://www.youtube.com/channel/UCjCpZyil4D8TBb5nVTMMaUw) and don't forget to click the subscribe button.

> > :warning: **These tutorials are written using Unity 2021.1 and what you see may have changed in future versions.**

Reviewer: Use admonitions. Also may just want it as important. Warnings are more for breaking, severe issues, etc.

```suggestion
:::important
These tutorials use Unity v2021.1. Future versions of Unity, Unity MLAPI, and Boss Room may have changes and updates to APIs and features.
:::
```

> This Unity Multiplayer Networking tutorial will teach you all about the new official Unity multiplayer solution and how to get it set up quickly in your own project!

Reviewer: Maybe a heading here?

```suggestion
## Make a Multiplayer Game

This Unity Multiplayer Networking tutorial will teach you all about the new official Unity multiplayer solution and how to get it set up quickly in your own project!
```

        allowfullscreen
        />

> This Unity Multiplayer tutorial will teach you how to implement connection approval so that clients have to send the correct password to be able to connect to the server.

Reviewer: Perhaps add a heading, especially if there may be more entries? 

```suggestion
## Lobby Password

This Unity Multiplayer tutorial will teach you how to implement connection approval so that clients have to send the correct password to be able to connect to the server.
```
